src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -86,6 +86,12 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Condition="'$(ControlFlowGuard)' == 'Guard'" Include="/guard:cf" />
     </ItemGroup>
 
+    <ItemGroup Condition="!Exists('$(IlcSdkPath)debugucrt.txt')">
+      <!-- Force ucrt to be dynamically linked for release runtime -->
```

Comment:
Mirrors https://github.com/dotnet/runtime/blob/main/eng/native/configurecompiler.cmake#L104-L106

We do not do this for debug CRT since it is pain to deploy (debug CRT is not part of the OS).

Reply:
I was scratching my head why we do this instead of passing `/MT`/`/MD` to the `cl` command line when building the object files.

The explanation I came up with is that we do this because we want to link libcmt.lib and libvcruntime.lib statically (like /MT does), but the parts that are in UCRT should be dynamic. That way we avoid a runtime dependency on VCRUNTIME140.dll. Is my understanding correct?

Reply:
Yes, that's correct. VCRUNTIME140.dll is tiny and linking it statically is much easier than figuring out how to deploy it in all scenarios.